crates/bevy_ecs/src/query/state.rs

```diff
@@ -398,6 +409,29 @@ impl<D: QueryData, F: QueryFilter> QueryState<D, F> {
         }
     }
 
+    // TODO: Better explanation, I'm not a fan of the term "reffed".
+    /// Gets the query result in "reffed" form (whatever can go in the [`Ref`] smart pointer - goes into it,
+    /// whatver doesn't - turns into the read-only version) for the given [`World`] and [`Entity`].
+    ///
+    /// This can only be called for read-only queries, see [`Self::get_mut`] for write-queries.
```

Comment:
I don't think this should be required: can we call `QueryState::get` on write-queries?

Reply:
I copied it from `QueryState::get`, I thought the same thing but wasn't sure.